server/http_handler_test.go

```diff
@@ -987,17 +988,41 @@ func (ts *HTTPHandlerTestSuite) TestDebugZip(c *C) {
 	resp, err := http.Get("http://127.0.0.1:10090/debug/zip?seconds=1")
 	c.Assert(err, IsNil)
 	c.Assert(resp.StatusCode, Equals, http.StatusOK)
-	out, err := os.Create("/tmp/tidb_debug.zip")
+	b, err := httputil.DumpResponse(resp, true)
 	c.Assert(err, IsNil)
-	_, err = io.Copy(out, resp.Body)
-	c.Assert(err, IsNil)
-	fileInfo, err := out.Stat()
+	c.Assert(len(b), Greater, 0)
+	c.Assert(resp.Body.Close(), IsNil)
+}
+
+func (ts *HTTPHandlerTestSuite) TestZipInfoForSQL(c *C) {
+	ts.startServer(c)
+	defer ts.stopServer(c)
+
+	db, err := sql.Open("mysql", getDSN())
+	c.Assert(err, IsNil, Commentf("Error connecting"))
+	defer db.Close()
+	dbt := &DBTest{c, db}
+
+	dbt.mustExec("use test")
+	dbt.mustExec("create table if not exists t (a int)")
+
+	urlValues := url.Values{
+		"sql":        {"select * from t"},
+		"current_db": {"test"},
+	}
+	resp, err := http.PostForm("http://127.0.0.1:10090/debug/sub-optimal-plan", urlValues)
 	c.Assert(err, IsNil)
-	c.Assert(fileInfo.Size(), Greater, int64(0))
-	err = out.Close()
+	c.Assert(resp.StatusCode, Equals, http.StatusOK)
+	b, err := httputil.DumpResponse(resp, true)
 	c.Assert(err, IsNil)
-	err = os.Remove("/tmp/tidb_debug.zip")
+	c.Assert(len(b), Greater, 0)
+	c.Assert(resp.Body.Close(), IsNil)
+
+	resp, err = http.PostForm("http://127.0.0.1:10090/debug/sub-optimal-plan?pprof_time=5&timeout=0", urlValues)
 	c.Assert(err, IsNil)
```

Comment:
add more test for pprof_time & timeout & urlValues .  like 
1. only use single pprof_time or single timeout
2. set wrong 'curDB'